realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncedRealmTests.kt

```diff
@@ -510,10 +507,45 @@ class SyncedRealmTests {
     }
 
     @Test
-    fun allTypes_roundTrip() {
+    fun catch_float32availableInSync(){
         val user1: User = createNewUser()
         val config1: SyncConfiguration = createDefaultConfig(user1, partitionValue)
 
+        Realm.getInstance(config1).use { realm1 ->
+            realm1.executeTransaction {
+                val syncObject = SyncAllTypes().apply {
+                    columnMixed = Mixed.valueOf(10.5.toFloat())
+                    columnRealmInteger.set(10)
+                }
+
+                realm1.copyToRealmOrUpdate(syncObject)
+            }
+
+            try {
+                realm1.syncSession.uploadAllLocalChanges()
+                fail("Sync now support float32 values.")
+            } catch (ignore: AppException) {
+
+            }
+        }
+    }
+
+
+    @Test
+    fun allTypes_roundTrip() {
+        val expectedMixedValues = arrayListOf(
+                Mixed.valueOf(1.toLong()),
+                Mixed.valueOf(false),
+//              Float not supported in sync
+//                Mixed.valueOf(10.5.toFloat()),
```

Comment:
dead code

Reply:
Sync will support float32 at some point, that's why I left that piece of unborn code. (we have a similar comment in 
 `SyncAllTypes.kt`)